src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -559,8 +559,7 @@
                                                                FontSize="12"
                                                                Margin="16,8,0,8">
                                                             <Run Text="{Binding Prefix, Mode=OneWay}"
-                                                                 MouseDown="OnPrefixTextBlockMouseDown" />
-                                                            <Run Text="{Binding GroupName, Mode=OneWay}">
+                                                                 MouseDown="OnPrefixTextBlockMouseDown" /><Run Text="{Binding GroupName, Mode=OneWay}">
```

Comment:
@Benglin , do you remember task, when I had to change inputs for tooltip. And `Run` had some default margin, that I couldn't remove. So, I changed 2 Runs in 2 TextBlocks. Now I know, why `Run` had this space. Answer is [here](http://stackoverflow.com/questions/11090084/how-to-get-rid-of-whitespace-between-runs-in-textblock).